ironfish/src/node.ts

```diff
@@ -49,7 +49,6 @@ export class IronfishNode {
   files: FileSystem
   rpc: RpcServer
   peerNetwork: PeerNetwork
-  syncer: Syncer
```

Comment:
The syncer is a top level concept to me, so it's weird for me to see this being demoted.